.github/workflows/backend.yml

```diff
@@ -0,0 +1,110 @@
+name: backend-py3
```

Comment:
I'd add a `test-` prefix somewhere. Not all workflows are tests 😉

Reply:
@billyvg your opinion on this? We should be consistent if I did make this change.

Reply:
FYI there's already a `release.yml` workflow.

Reply:
Yeah, I'm talking about the others, they should be renamed too.

Reply:
I've been trying to group workflows based on the `on` event. Does this need to  run if only javascript files are changed? I'd move it into the `python` workflow and name the job as `test-py3`

Reply:
No to javascript. And ok, I'll look into that.